keyboards/projectd/75/iso/keymaps/default/config.h

```diff
@@ -0,0 +1,20 @@
+/* Copyright 2023 GSKY <gskyGit@gsky.com.tw>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
```

Comment:
Is the locking supported needed?   If not, then this whole file could be removed.

Otherwise, needs an include guard: 
```suggestion
#pragma once

```

Reply:
> Is the locking supported needed? If not, then this whole file could be removed.
> 
> Otherwise, needs an include guard:

@drashna Before it has locking function in config.h file but it moved to keymap folder now. please let me where I need to keep that function, thanks.
![image](https://github.com/qmk/qmk_firmware/assets/106651989/817ca1cd-bcb9-44df-81a0-ee9a9cb69fa5)